mlrun/serving/states.py

```diff
@@ -783,6 +783,7 @@ def insert_step(self, key, step, after, before=None):
                 raise GraphError(
                     f"graph loop, step {before} is specified in before and/or after {key}"
                 )
+            self[step.name].after_step(*self[before].after, append=False)
```

Comment:
what issue is it solving?

Reply:
Correctly defines the position of the new step in the graph. 
A step contains only the after parameter, so if we don't define it here, the step we entered will be disconnected from the beginning of the graph.